plugin/src/index.ts

```diff
@@ -41,22 +36,21 @@ function getGenerator(options: PluginOptions) {
 
 function plugin(_options?: PluginOptions): Plugin[] {
   let env: ConfigEnv;
-  let resolvedConfig: ResolvedConfig;
   const resolvedDeps: Set<string> = new Set();
   const installPromiseCache: Promise<unknown>[] = [];
 
   return [
     {
       name: "jspm:imports-scan",
       enforce: "pre",
-      config(_config, _env) {
+      config(_, _env) {
         env = _env;
       },
-      configResolved(_config) {
-        resolvedConfig = _config;
+      configResolved(config) {
+        config.build.polyfillModulePreload = false;
```

Comment:
What about DynamicImport?

Reply:
Looks like it is removed from the vite build.options itself. It isn't configurable anymore

Reply:
Ok, then, please add that note to the docs so people in older versions of vite are informed to consider that option.

Reply:
Actually, that not mandatory. Because, `es-module-shims` can detect these and disable it's own polyfills. And adds for only missing feature. It just adds confusion for the users, to check which version of `vite` they are using etc.